app/views/articles/index.html.haml

```diff
@@ -0,0 +1,10 @@
+%h2 Articles
+
+- @articles.each do |article|
+    %img{:src => "/assets/article.jpg"}/
```

Comment:
Please use `image_tag` helper to display images

Reply:
Done